etcd_helper/tests/etcd_test.go

```diff
@@ -0,0 +1,80 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/danielpygo/kubeplus/etcd_helper"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	rdr  etcd_helper.Etcdreader
+	wrtr etcd_helper.Etcdwriter
+)
+
+func init() {
+	rdr.EtcdServiceURL = "http://localhost:2379"
+	wrtr.EtcdServiceURL = "http://localhost:2379"
+}
+
+func TestGet(t *testing.T) {
+	err := wrtr.Store("postgres", "THIS IS SOME DATA")
+	if err != nil {
+		fmt.Printf("Err: %s\n", err)
+	}
+	str, err := rdr.Get("postgres")
+	if err != nil {
+		fmt.Printf("Err: %s\n", err)
+	}
+	assert.Equal(t, str, "THIS IS SOME DATA", "A simple Post and Get of one item")
+}
+func TestGetList(t *testing.T) {
+	dataList := []string{"database1", "database2", "database3"}
+	err := wrtr.StoreList("postgres1", dataList)
+	if err != nil {
+		fmt.Printf("Err: %s\n", err)
+	}
+	strSlice, err := rdr.GetList("postgres1")
+	if err != nil {
+		fmt.Printf("Err: %s\n", err)
+	}
+	assert.Equal(t, strSlice, dataList, "A simple Post and Get of an array of dataItems")
+}
+
+func TestDeleteFromList(t *testing.T) {
```

Comment:
Can you add one more test where dataList := []string{"database1"}

wrtr.StoreList("postgres2", dataList)
wrtf.Delete("postgres2", "database1")

In this case the key should get deleted since the only item in the list is being deleted.